lib/Timber.php

```diff
@@ -439,22 +439,32 @@ public static function get_sidebar( $sidebar = 'sidebar.php', $data = array() )
 	 * @return string
 	 */
 	public static function get_sidebar_from_php( $sidebar = '', $data ) {
-		$caller = LocationManager::get_calling_script_dir(1);
-		$uris = LocationManager::get_locations($caller);
+		$caller = LocationManager::get_calling_script_dir( 1 );
+		$uris   = LocationManager::get_locations( $caller );
 		ob_start();
 		$found = false;
-		foreach ( $uris as $uri ) {
-			if ( file_exists(trailingslashit($uri).$sidebar) ) {
-				include trailingslashit($uri).$sidebar;
-				$found = true;
-				break;
+		foreach ( $uris as $namespace => $uri_locations ) {
+			if ( is_array( $uri_locations ) ) {
+				foreach ( $uri_locations as $uri ) {
+					if ( file_exists( trailingslashit( $uri ) . $sidebar ) ) {
+						include trailingslashit( $uri ) . $sidebar;
+						$found = true;
+					}
+				}
+			} else {
+				if ( file_exists( trailingslashit( $uri_locations ) . $sidebar ) ) {
+					include trailingslashit( $uri_locations ) . $sidebar;
+					$found = true;
+					break;
+				}
```

Comment:
I'm being overly defensive here as `$uri_locations ` will always be an array, as it affects code coverage I'll remove this if/else.